### ISSUE TITLE
Models self-deleting?

### ISSUE THREAD

**Issue by AlexBeez** [cocktailpeanut/dalai#333]
On occasion, the model appears to self-destruct, delivering the message that "file does not exist". 

Anyone encounter that problem before?

**Comment by RexBarker** [comment]
Yes, same for me.  It was working fine on the weekend.  When I started it yesterday, all models were gone and I had other errors.  Not sure why this can't work in a more stable dev/release type pattern
